Defuq/Assets/Scripts/ScriptableObjectScripts/EnemyStatsSO.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+
+[CreateAssetMenu(fileName = "Enemy",menuName = "Create New Enemy SO",order = 1)]
+public class EnemyStatsSO : ScriptableObject
```

Comment:
You already got this script in another folder, did you forgot to delete this one?

Reply:
where?

Reply:
I only have one scriptable object script...